astromon/cross_match.py

```diff
@@ -747,6 +747,7 @@ def simple_cross_match(
     matches['dz'] = matches['x_z_angle'] - matches['c_z_angle']
     matches['dy'] = matches['x_y_angle'] - matches['c_y_angle']
     matches['dr'] = np.sqrt(matches['dy']**2 + matches['dz']**2)
+    matches['cat_order'] = table.Column(length=len(matches), dtype=int)
```

Comment:
You can do with in one line with:
```
matches['cat_order'] = np.full(len(matches), 200)
```
With modern astropy the `Column` class constructor is almost never needed for user code.

Reply:
I adopted this suggestion.